web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table.html.twig

```diff
@@ -40,10 +40,10 @@
     sticky ? 'sticky-header',
     'table-auto',
     'w-full border-separate border-slate-400',
+    'min-w-[1000px]',
```

Comment:
Will this work on mobile?

Reply:
The purpose is mobile with overflow-x-auto class. To force a scrollbar when the table becomes too small

Reply:
Ahh - I see. Good work.